docs-ui/components/formatters.stories.js

```diff
@@ -100,4 +102,28 @@ storiesOf('Utility|Formatters', module)
         </div>
       </div>
     ))
+  )
+  .add(
+    'Version',
+    withInfo('Formats release version')(() => {
+      const version = text('version', 'foo.bar.Baz@1.0.0+20200101');
+      const orgId = text('orgId', 'sentry');
+      const anchor = boolean('anchor', true);
+      const preserveGlobalSelection = boolean('preserveGlobalSelection', false);
+      const tooltipRawVersion = boolean('tooltipRawVersion', true);
+      const className = text('className', 'asdsad');
+      return (
+        <div>
+          {version} =
```

Comment:
please, is this right?

Reply:
Yes, it is like a before/after thing. Other formatter called count does the same thing.
![image](https://user-images.githubusercontent.com/9060071/73658824-53e7e800-4695-11ea-8a44-afab8492a551.png)

Reply:
Interesting! Thanks for the clarification ;-)